shell/platform/tizen/flutter_tizen.cc

```diff
@@ -74,14 +74,10 @@ void FlutterDesktopEngineShutdown(FlutterDesktopEngineRef engine_ref) {
   delete engine;
 }
 
-void* FlutterDesktopPluginRegistrarGetNativeWindow(
```

Comment:
Apparently this API is never used. Why was this added in the first place? cc @bbrto21

Reply:
As far as I can remember, these were used in plugins, but over time they didn't use the handle from this at all, and eventually, everywhere that uses this API is gone recently.

Reply:
I think we can remove it in this PR or start deprecation step for it.
cc @bwikbs what do you think?

Reply:
@bbrto21 Will there be a plugin that the API uses? If you don't think it's going to happen, what do you think about deleting the API from this PR?

Reply:
According to my memory.. It's prepared for ewk(chromium webview)'s key event handling. 
and.. I'm not sure if it's still necessary or not. :sob:

Reply:
If you're not sure, how about leaving it as is?

Reply:
FlutterDesktopPluginRegistrarGetNativeHandle and FlutterDesktopViewGetNativeHandle return the same handle.
I think we can remove FlutterDesktopPluginRegistrarGetNativeHandle and use FlutterDesktopViewGetNativeHandle if necessary.

Reply:
Consider removing the API if it's unused, or redesign the API so that users can clearly understand the usage.

```cpp
// Bad:
// Returns the native handle of |registrar|?
handle = FlutterDesktopPluginRegistrarGetNativeHandle(registrar);

// Good:
view = FlutterDesktopPluginRegistrarGetView(registrar);
handle = FlutterDesktopViewGetNativeHandle(view);
```

Reply:
> Consider removing the API if it's unused, or redesign the API so that users can clearly understand the usage.

I think it would be better to treat this as a different topic. @swift-kim What do you think?

Reply:
> Consider removing the API if it's unused, or redesign the API so that users can clearly understand the usage.

@Swanseo0 This conversation still open, please don't forget handle it.

Reply:
```
view = FlutterDesktopPluginRegistrarGetView(registrar);
handle = FlutterDesktopViewGetNativeHandle(view);
```

Have removed the FlutterdesktopplugInegistrargetnativeHandle function and added the FlutterdesktopplugIngistrargetView function.

Unlike using the existing API, two steps are required, but it seems to be more intuitive for users to use.

Reply:
> Consider removing the API if it's unused, or redesign the API so that users can clearly understand the usage.
> 
> ```c++
> // Bad:
> // Returns the native handle of |registrar|?
> handle = FlutterDesktopPluginRegistrarGetNativeHandle(registrar);
> 
> // Good:
> view = FlutterDesktopPluginRegistrarGetView(registrar);
> handle = FlutterDesktopViewGetNativeHandle(view);
> ```

I think `FlutterDesktopViewGetNativeHandle` required by ewk_webview_flutter to show software keyboard.
@Swanseo0 please rebase this PR and make a related PR on flutter-tizen and open it.

Reply:
Also change the target branch to `flutter-3.0.5-tizen`.

Reply:
@bbrto21 @swift-kim 
After changing the target branch to flutter-3.0.5-tizen, I rebased this PR.
I haven't opened a PR related to flutter-tizen repo yet, but after this PR is merged, i check the engine hash and apply the hash to the relevant PR.
Is it okay to merge this PR? Or do PR need more modifications? Please tell me your opinion.

Reply:
Not okay.

1. Open the tool side PR first.
2. Review the PR.
3. Merge the engine side PR.
4. Update the tool side PR with the new engine hash.
5. Merge the tool side PR.

Reply:
1-1. Open the plugins PR for [this draft](https://github.com/flutter-tizen/plugins/pull/400) in @bwikbs's repository 
2-1. Review the plugins PR.
5-1. Merge the plugins side PR.

[This draft](https://github.com/flutter-tizen/plugins/pull/400) is constantly being tested for new tizen profile and you should not break the build status.